docker/Dockerfile

```diff
@@ -0,0 +1,110 @@
+FROM centos:latest
+
+# The Suricata version to install. This should match the version in the download
+# URL. E.g: https://www.openinfosecfoundation.org/download/suricata-<VERSION>.tar.gz
+ENV SURICATA_VERSION '4.0.5'
```

Comment:
you can use 'current' here and it will always pull the latest stable (suricata-current.tar.gz is kept in sync with last stable)

Reply:
Thanks, I've amended the Dockerfile to build from the source tree now that it's part of the repo.